app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt

```diff
@@ -128,7 +131,7 @@ fun LoginScreen(
                 OutlinedButton(
                     onClick = onForgotPasswordClick,
                     modifier = Modifier.align(Alignment.CenterHorizontally)
-                ) { Text(stringResource(R.string.auth_login_forgot_password)) }
+                ) { Text(stringResource(R.string.auth_login_forgot_password), color = MaterialTheme.colorScheme.onBackground) }
```

Comment:
Same as [with this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847357323). Since there are multiple parameters, it should be indented.